apps/studio/src/server/modules/resource/resource.router.ts

```diff
@@ -195,7 +195,6 @@ export const resourceRouter = router({
         .selectFrom("Resource")
         .select(["title", "permalink", "type", "id", "parentId"])
         .where("Resource.type", "!=", ResourceType.RootPage)
-        .where("Resource.type", "!=", ResourceType.IndexPage)
```

Comment:
question: hmm why did we remove this? `getChildrenOf` looks like it is used in `useResourceQuery` which is used in `ResourceSelector` as part of move items flow + adding links. I think for adding links, they should be linking to the folder rather than to the index page directly.

Reply:
this got removed because it's a bug fix. this method controls what's viewable from the sidebar, and we should do this in the subsequent method for `useResourceQuery`